### ISSUE TITLE
Compile Errors

### ISSUE THREAD

**Issue by TroyDanielFZ** [stevencohn/OneMore#12]
Sorry to bother you, but I did get some trouble on compiling this project.
I succeded in compiling Addin & Tests, but the Setup is incompatible with my IDE(VS2017 Community, with recent upgrade). I'm new in developing a distributable project, so I didn't get any solutions even if I'd tried surfing the Internet.
I'm wandering whether this project has any prerequisites or not. And it'll be greatful if you can give me a hand.

**Comment by stevencohn** [comment]
Hi, I don't have Community edition installed. Can you send me the compile log? Otherwise, I may be able to test it out in a virtual machine.

**Comment by TroyDanielFZ** [comment]
I'm so sorry that I reply so late.
But I try other repository named ‘OneMore', and I succeed without knowing why, because I tried so many ways.
Today, I re-download this repository and compile it again, I get the error that I haven't get 'System.ValueTuple`4' installed. So I've tried to download this with Nutget Package Manger, and I suceeded in compiling and installing this plugin.

Thanks for your great work!

**Comment by stevencohn** [comment]
Yes, the System.ValueTyule Nuget package is required by this solution. It is specified in the packages.config file. I've noticed sometimes that when trying to build a fresh clone of the repo without first "Restoring Nuget Packages" then the build may fail. VS should do this restore step first but sometime it appears it does not. I got in the habit of always manually choosing "Restore Nuget Packages" before I run Build. I hope this helps.

**Comment by TroyDanielFZ** [comment]
Thank you for your patience.
And here comes new problems:

I don't know why this plugin doesn't work properly right now, Today, I tried to make some changes to the code, but I found whatever I did to the code, the behaviour didn't change. After a lot of debugging, I find that I didn't unitall fully of my initial trying of your binary, maybe version 1.5.5. So I UNISTALLED everythin plugins of onenote and deleted the files in the folder. Sadly, after this step, I cannot install onemore properly even if with your binary contribution of version 1.5.7. I find trhee files in the installation folder:
``
license.rtf
River.OneMoreAddIn.dll
System.ValueTuple.dll
``
and then, within onenote, no onemore appears in the ribbon, and in "options>addins", I get OneMoreAddin in "inactive application Addin" with position "mscore.dll".
![default](https://user-images.githubusercontent.com/31643818/40848258-3f82af2a-65f1-11e8-92d9-7a0efeb651f0.png)

I don't know how to sovle this, did you come up with this problem once?

**Comment by stevencohn** [comment]
If you click the Go button on that dialog (next to the COM drop-down near the bottom) then you can Remove the addin. Microsoft recommends doing this prior to uninstalling an add-in.

If you intend on developing and debugging the add-in then you can change the registration path to be the output folder of the OneMoreAddIn project rather than re-installing on every change. So just run out of OneMore\OneMoreAddIn\bin\x86\Debug. Then when you make a change, you only need to close OneNote, recompile, and then restart OneNote. Save the installer for later when you know you're done.

**Comment by TroyDanielFZ** [comment]
I've tried, but I get the error message that "OneMoreAddIn\bin\x86\Debug\River.OneMoreAddIn.dll" isn't a valid Office addin. The steps I took are:
1. Click button "Go" near the  Com drop-down;
2. Choose OneMOre Addin  & delete it;
3. Close OneMore & restart it;
4. Press button "Add" & redriect to the debugging folder & choose "River.OneMoreAddIn.dll";
5. I get the error message mentioned above.

PS: I used onenote x64 on Windows 10, does this cause the problem? (I changed the target platform into x64, but it doesn't work actually.)
